WordPress/gutenberg
# Make inline toolbar navigable by arrow keys

## What?


Allows users to navigate the inline toolbar using arrow keys rather than tabs.

## Why?


Users can navigate the block toolbar using arrow keys. The inline toolbar should behave the same way for consistency.

Resolves #43564

## How?


Adds the `NavigableToolbar` component to the inline toolbar.

## Testing Instructions





1. Add an Image block to the page, and insert an image in it;
1. Caption should be focused automatically;
1. Press Shift+Tab to enter the caption toolbar;
1. Note that the toolbar can be navigated using arrow keys.
1. Press Tab and note that focus moves back to the caption.
